api/app/controllers/review.controller.ts

```diff
@@ -581,3 +432,99 @@ export const resetReviews = async (req: Request, res: Response) => {
     res.status(500).send(`Error resetting review: ${err}`);
   }
 };
+
+
+
+// utility functions.
+
+const buildCondition = (
```

Comment:
This can be moved to the [utils](https://github.com/bitcointranscripts/transcription-review-backend/tree/main/api/app/utils) folder inside of `review.inference.ts`

Reply:
Also, I think its easier if we use object args for this so we can explicity define the arguments where the function is used
```js
const buildCondition = ({
  status,
  transcriptId,
  userId,
  mergedAt,
  userSearch
}:{
  status?: string,
  transcriptId?: number,
  userId?: number,
  mergedAt?: string,
  userSearch?: string
})
```